File: 프로그래머스/unrated/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.py
def solution(my_string):
    #합계를 저장할 변수를 초기화
    total = 0
    
    #문자열 my_string을 순회하면서 각 문자를 확인합니다.
    for char in my_string:
        if char.isdigit():
            
            total += int(char)
    return total

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        // 문자열 charAt으로 하나하나 isDigit확인하기 맞으면
        //answer에 더함
        for(int i=0;i<my_string.length();i++){
            if(Character.isDigit(my_string.charAt(i))){
                answer += Character.getNumericValue(my_string.charAt(i));
            }
        }
        
        return answer;
    }
}